concepticondata/conceptlists/RonaTas-2011-431.tsv

```diff
@@ -0,0 +1,432 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+RonaTas-2011-431-1	1	carpenter	1042	CARPENTER
+RonaTas-2011-431-2	2	cannon, catapult	3202	CANNON
+RonaTas-2011-431-3	3	to fondle, pet, caress	3780	PET
+RonaTas-2011-431-4	4	stream, current	2935	CURRENT (STREAM)
+RonaTas-2011-431-5	5	false, imitation	3069	FALSE
+RonaTas-2011-431-6	6	pied (of an animals coat)		
+RonaTas-2011-431-7	7	apple	1320	APPLE
+RonaTas-2011-431-8	8	barrel (wooden container andstandard quantity)	2548	BARREL
+RonaTas-2011-431-9	9	to nurse, take care of	2472	SUCKLE (BREASTFEED, NURSE)
+RonaTas-2011-431-10	10	to decay, putrify, turn stale	2907	ROT
+RonaTas-2011-431-11	11	small, tiny	1246	SMALL
+RonaTas-2011-431-12	12	to mow	190	MOW
+RonaTas-2011-431-13	13	to decay, putrefy, go bad (of food, water)	2907	ROT
+RonaTas-2011-431-14	14	to harm	44	INJURE
+RonaTas-2011-431-15	15	a type of lasso		
+RonaTas-2011-431-16	16	ditch, canal	2002	DITCH
+RonaTas-2011-431-17	17	barley, Hordeum	932	BARLEY
+RonaTas-2011-431-18	18	barrow	2565	CASTRATED BOAR
+RonaTas-2011-431-19	19	gauntry, gantry (for supporting barrels)		
+RonaTas-2011-431-20	20	charm		
+RonaTas-2011-431-21	21	soggy place, moor, swamp, marsh	1145	SWAMP
+RonaTas-2011-431-22	22	hatchet, axe	677	AXE
+RonaTas-2011-431-23	23	pagan idol, stone or wooden pillar, vertical beam of different wooden instruments, strong, corpulent person	857	STONE
```

Comment:
`STONE` refers to english "stone", while this entry seems to be `1945 IDOL`. I'd change the mapping.

Reply:
resolved in 3288105d13e2264f434aa08790bfaaebcd5e3458